Makefile

```diff
@@ -176,7 +176,8 @@ INCLUDES		+=	-Iinclude/bl1				\
 				-Iinclude/plat/common			\
 				-Iinclude/services			\
 				${PLAT_INCLUDES}			\
-				${SPD_INCLUDES}
+				${SPD_INCLUDES}				\
+				-Iinclude/host_api
```

Comment:
I fixed up and sent a new pull instead of forced push
because this pull-request includes host_api in its branch name and subject.